cmd/ocm/cluster/list/list.go

```diff
@@ -142,6 +148,7 @@ func run(cmd *cobra.Command, argv []string) error {
 
 	size := 100
 	index := 1
+	managed := args.managed
```

Comment:
I don't think this will do what you think it will. See https://github.com/openshift-online/ocm-cli/pull/81